src/main/kotlin/uk/gov/dluhc/printapi/database/entity/AnonymousElectorDocument.kt

```diff
@@ -41,40 +41,40 @@ class AnonymousElectorDocument(
 
     @field:NotNull
     @field:Size(max = 20)
-    var certificateNumber: String? = null,
+    var certificateNumber: String,
 
     @field:NotNull
     @Enumerated(EnumType.STRING)
-    var sourceType: SourceType? = null,
+    var sourceType: SourceType,
 
     @field:NotNull
     @field:Size(max = 255)
-    var sourceReference: String? = null,
+    var sourceReference: String,
 
     @field:Size(max = 255)
-    var applicationReference: String? = null,
+    var applicationReference: String?,
 
     @field:NotNull
     @Enumerated(EnumType.STRING)
-    var certificateLanguage: CertificateLanguage? = null,
+    var certificateLanguage: CertificateLanguage?,
 
     @Enumerated(EnumType.STRING)
-    var supportingInformationFormat: SupportingInformationFormat? = null,
+    var supportingInformationFormat: SupportingInformationFormat?,
 
     @field:NotNull
     @field:Size(max = 255)
-    var photoLocationArn: String? = null,
+    var photoLocationArn: String,
```

Comment:
Similar comment for as the above one. Seems like for the `VAC`s we have `photoLocationArn` in the `PrintRequest` entity, rather than the `Certificate`, possibly allowing different `photoLocationArn`s for different print requests  of the same application. 

Is that not the case for the `AED`s? 

Can we not have two different `photoLocationArn`s for two different print requests for the same `AED` application?

Reply:
As above.